lib/tasks/wayback.rake

```diff
@@ -0,0 +1,35 @@
+desc 'Request the Wayback Machine to snapshot a specific URL or all new '\
+  'article URLs for the past 24 hours if no argument is given'
+
+task :wayback, [:url] => :environment do |_t, args|
+  app = JSON.parse(File.read(Rails.root.join('app.json')))
```

Comment:
i think it is fine to hardcode the website here, can't really think of a situation where it would change

Reply:
Either way is fine by me. I'ma merge this as is. @explody if you want to hardcode this in another PR, cool.